tasks/main.yml

```diff
@@ -2,6 +2,12 @@
 - name: Include OS-specific variables.
   include_vars: "{{ ansible_os_family }}.yml"
 
+- name: "Include variables for Amazon Linux. {{ ansible_distribution }}  {{ ansible_distribution_major_version }}"
```

Comment:
It looks like there are a couple debug things left in this `name` (distribution and major version) — can those be removed?